packages/cosmos-client/src/CosmosClient.ts

```diff
@@ -0,0 +1,43 @@
+import { map } from "rxjs/operators";
+import { request, RequestResponse } from "@dcos/http-service";
+
+import Config from "#SRC/js/config/Config";
+import { buildRequestHeader, getErrorMessage } from "./utils";
+
+import { PackageVersionsResponse } from "../";
+
+export const CosmosClient = {
+  listPackageVersions: (packageName: string) =>
+    request<PackageVersionsResponse>(
+      `${Config.rootUrl || "/"}package/list-versions`,
+      {
+        method: "POST",
+        body: JSON.stringify({
+          includePackageVersions: true,
+          packageName
+        }),
+        headers: {
+          Accept: buildRequestHeader(
+            "list-versions",
+            "response",
+            "package",
+            "v1"
+          ),
+
```

Comment:
🙈

Reply:
I hope you meant that there's a newline.